charts/kong/README.md

```diff
@@ -203,50 +203,45 @@ Kong can be configured via two methods:
 
 ### Kong parameters
 
+The various `SVC.*` parameters below are common to the various Kong services
+(the admin API, proxy, Kong Manger, the Developer Portal, and the Developer
+Portal API) and define their listener configuration, K8S Service properties,
+and K8S Ingress properties. Defaults are listed only if consistent across the
+individual services: see values.yaml for their individual default values.
+
```

Comment:
1. Let's split this out and create a new section in the readme with a table dedicated to these `svc.*` configuration.
2. Let's be very clear about what `svc` can be substituted by else this will trigger a lot of support request. I'd also recommend replacein `SVC` with `{SVC}` or some other notation to indicate that it needs to be substituted. Something more polished like:

`{SVC}` in the below table can be substite by:
- `proxy`
- `admin`
- `manager`
- and so on